Tests/SentryTests/Protocol/SentrySDKSettings+Equality.m

```diff
@@ -0,0 +1,30 @@
+#import "SentrySDKSettings+Equality.h"
+
+@implementation SentrySDKSettings (Equality)
+
+- (BOOL)isEqual:(id _Nullable)object
+{
+    if (object == self)
+        return YES;
+    if ([self class] != [object class])
+        return NO;
+
+    SentrySDKSettings *otherSDKSettings = (SentrySDKSettings *)object;
+
+    if (!self.autoInferIP == otherSDKSettings.autoInferIP) {
+        return NO;
+    }
+
+    return YES;
+}
+
+- (NSUInteger)hash
+{
+    NSUInteger hash = 18;
```

Comment:
`l`: I might be wrong here, but shouldn't hashing functions use prime numbers to reduce the chance of collisions?

```suggestion
    NSUInteger hash = 17;
```